README.md

```diff
@@ -4,9 +4,113 @@
 
 The Helm Broker is a [Service Broker](https://kyma-project.io/docs/master/components/service-catalog/#service-brokers-overview) which exposes Helm charts as Service Classes in the [Service Catalog](https://kyma-project.io/docs/master/components/service-catalog/#overview-overview). To do so, the Helm Broker uses the concept of addons. An addon is an abstraction layer over a Helm chart which provides all information required to convert the chart into a Service Class.
 
-For more information, read the [Helm Broker documentation](https://kyma-project.io/docs/master/components/helm-broker/).
+For more information, read the [Helm Broker documentation](https://kyma-project.io/docs/master/components/helm-broker/). If you need Helm-Broker installed with all dependencies try [Kyma](https://kyma-project.io/).
 
-## Prerequisites
```

Comment:
Please turn back this section. List the requirements to run the project and provide links.